core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -39,6 +39,12 @@ type ContributionOpportunityParams = {
   language_code?: string;
 };
 
+export interface IFetchOpportunitySuccessCallbackParams {
+  opportunities?: SkillOpportunity[],
```

Comment:
Why these are marked as optional?

Reply:
@DubeySandeep That shouldn't be there, glad that you mentioned!

Reply:
@h-ARTS, is this comment resolved?

Reply:
@DubeySandeep This is resolved. PTAL in file `contribution-opportunities.service.ts`